js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -58,6 +64,12 @@ export default function SetupPaidAds() {
 		await finishFreeListingsSetup( event );
 	};
 
+	// TODO: Add more disabled conditions to check
+	// - Google Ads account connection
+	// - Campaign data
+	// - Billing setup
+	const disabledComplete = completing === 'skip-ads';
```

Comment:
💅  Id use this skip-ads as a constant SKIP_ADS

Reply:
As per the suggestion in https://github.com/woocommerce/google-listings-and-ads/pull/1649#discussion_r957291065, I have added constants in https://github.com/woocommerce/google-listings-and-ads/pull/1655/commits/0e65308cf6ecb3163adadd39530172ffdaf8ef81 and https://github.com/woocommerce/google-listings-and-ads/pull/1655/commits/c08364ff753c5a2c50096e5b2254fd0c9ded0032.